### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by dieg0varela**
Me and Raimundo add these functions , if someone can check it and tell us if we need to change something it will be amazing

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,21 @@ def delete(self, obj=None):
...
+        """Return an object with a given class and ID"""
+        ret = None
+        objects = self.all(cls)
+        lent = len(cls.__name__) + 1
+        for i in objects:
+            if i[lent:] == id:
+                ret = objects[i]
+        return (ret)
+
+    def count(self, cls=None):
```
Comment (coding-max) [inline models/engine/file_storage.py:83]: I think it is a good idea to take into account the case that the class is not valid.
Example: `count(Country)`

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,21 @@ def reload(self):
...
+            if i[lent:] == id:
+                ret = objects[i]
+        return (ret)
+
+    def count(self, cls=None):
+        """Count the amount of objects of a class or all objects"""
+        ret = 0
+        objects = self.all(cls)
+        for i in objects:
+            ret += 1
```
Comment (coding-max) [inline models/engine/db_storage.py]: you can replace the lines 91-93 for: `ret = len(self.all(cls))`

**Review by coding-max — CHANGES_REQUESTED** [review]

File: models/engine/file_storage.py (hunk 1 above)
Comment (RaimundoGallino) [reply models/engine/file_storage.py:83]: Thats a good example of a border case that our project does not handle. Thanks for the request! @coding-max

File: models/engine/db_storage.py (hunk 2 above)
Comment (RaimundoGallino) [reply models/engine/db_storage.py]: Oh that's clever! A simple but efficient way of cycling through all instances!

File: models/engine/file_storage.py (hunk 1 above)
Comment (coding-max) [reply models/engine/file_storage.py:83]: It is always a good option to take into account the edge cases, remember that you have to do unitest for this project and using unexpected arguments should always be taken into account :)

**Pull request merged**